server/.env.example

```diff
@@ -4,6 +4,7 @@ DATABASE_URL=airshipper.db
 AIRSHIPPER_GITLAB_SECRET=banana
 AIRSHIPPER_ARTIFACT_STAGE=post
 AIRSHIPPER_TARGET_BRANCH=master
+#AIRSHIPPER_TARGET_VARIABLE=nightly
```

Comment:
not needed to comment that out. It's an example anyway to showcase possible values for these.